new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -551,11 +588,48 @@ public static JsonObject createDockerConfigJson(String username, String password
    * @throws InterruptedException if any thread has interrupted the current thread
    */
   public static ExecResult execCommand(V1Pod pod, String containerName, boolean redirectToStdout,
-      String... command)
+                                       String... command)
       throws IOException, ApiException, InterruptedException {
     return Exec.exec(pod, containerName, redirectToStdout, command);
   }
 
+  // ----------------------   pod  ---------------------------------
+
+  /**
+   * Get the creationTimestamp for a given pod with following parameters.
+   *
+   * @param namespace namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
```

Comment:
could use a link to the k8s doc here to explain this format

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/